recOrder/plugin/gui.py

```diff
@@ -1136,7 +1136,7 @@ def setupUi(self, Form):
         self.gridLayout_7.addWidget(self.label_logo, 0, 0, 1, 1)
 
         self.retranslateUi(Form)
-        self.tabWidget.setCurrentIndex(1)
+        self.tabWidget.setCurrentIndex(0)
```

Comment:
What does this do?

Reply:
Changes the startup tab to the calibration tab (as desired). 

QT Creator will set the default tab to whatever tab you have open when you save the file, so this is reverting an accidental change I introduce.d